vere/http.c

```diff
@@ -2554,45 +2579,52 @@ _proxy_parse_sni(const uv_buf_t* buf_u, c3_c** hot_c)
   return u3_pars_good;
 }
 
-/* _proxy_parse_ship(): determine destination for proxied request
+/* _proxy_parse_ship(): determine destination (unit ship) for proxied request
```

Comment:
I think ideally at some point we'd want to output some diagnostic information if this fails so if someone starts spamming us with invalid requests we'll know. Probably not actionable for now.

Reply:
Agreed. Note that if the `(unit ship)` is null, we attempt to handle the request ourselves. This is to support both custom domains and proxying (but not proxying over custom domains).